ads.properties.sample

```diff
@@ -0,0 +1,49 @@
+# OAuth refresh token
+#
+# Credential which authorizes access to a Google account. 
+#
+# - See guide for instructions on how to obtain a refresh token.
+#   https://developers.google.com/google-ads/api/docs/oauth/client-library
+# - Can access many Google Ads accounts. 
+# - Operating account is specified by the customerId field in each request. 
+#api.googleads.refreshToken=
+
```

Comment:
We should add:

```
#api.googleads.loginCustomerId=
```

I would think a significant portion of users use credentials from a single manager account and just want to set and forget this value.

Reply:
We already have a section for LCID, PTAL.

Reply:
Sorry, don't know how I missed that...